spec/support/geocoder_stubs.rb

```diff
@@ -39,3 +39,13 @@
     },
   ]
 )
+
+Geocoder::Lookup::Test.add_stub(
```

Comment:
Intended to include?

Reply:
Yeah, I'm always having to stick this in to get my tests to pass.